src/Boundaries/Sediments/coupled_timesteppers.jl

```diff
@@ -44,6 +46,25 @@ import Oceananigans.TimeSteppers: ab2_step!, rk3_substep!
     return nothing
 end
 
+@inline function ab2_step!(model::HydrostaticFreeSurfaceModel{<:Any, <:Any, <:AbstractArchitecture, <:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:Any, <:ContinuousFormBiogeochemistry{<:Any, <:FlatSediment}}, Δt, χ)
+    # Step locally velocity and tracers
+    @apply_regionally local_ab2_step!(model, Δt, χ)
+
+    # blocking step for implicit free surface, non blocking for explicit
```

Comment:
What does this mean?

Reply:
No idea, this is copied from the Oceananigans repo.

I don't think this is the ideal solution but the only was I can currently think of to integrate the sediment fields.

Reply:
Hm. My question was about the comment. Perhaps is a leftover comment from KernelAbstractions v0.8? Is this comment still in Oceananigans?

Reply:
Yeah:
https://github.com/CliMA/Oceananigans.jl/blob/56fc69bcb08623424afe25e39ae616c29208095e/src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_ab2_step.jl#L11C1-L20C4

Reply:
Ok!